README.md

```diff
@@ -239,6 +240,92 @@ This already goes with how CSP is currently enforcing its rules canonically in t
 
 `X-frames` and `frame-src` allow controlling what is allowed to be loaded into an `iframe`. Regardless of their use to deny loading content in iframes, ways to crete new same origin realms remain available, such as creating an `about:blank` iframe, opening a new tab using `open` API and more.
 
+## Consideration
+
+Some important points to consider
+
+### Privacy
+
+There isn't much to take into consideration in regards with privacy.
+
+### Security
+
+Here are some security risks to take into consideration when following this spec for implementation purposes:
+
+#### Universal XSS
+
+Since this proposal focuses on the thin line between same and cross origins, it may put SOP in danger if not implemented correctly.
```

Comment:
I think it'd be good to rephrase this and clarify.
"""
#### XSS risk

This proposal enables and top-level document to execute remote scripts in same-origin realms.

If implementations somehow enable that execution to happen in cross-origin realms, that can enable cross-origin scripting in those realms.

Therefore it's important for implementations to get that part right, and for tests to throughly cover that possibility. 
This is similar to other powerful features that heavily rely on the same-origin policy, such as Service Workers.
"""

This also makes me think that we'd want to restrict this feature to secure contexts.

Reply:
90036e9acf89b3185ef5fef0812d3966116fca2a